packages/loot-design/src/components/modals/ImportTransactions.test.js

```diff
@@ -0,0 +1,110 @@
+import { parseDate } from './ImportTransactions';
+
+describe('Import transactions', function() {
+  describe('date parsing', function() {
+    it('should not parse', function() {
+      const invalidInputs = [
+        { str: '', order: 'yyyy mm dd' },
+        { str: null, order: 'yyyy mm dd' },
+        { str: 42, order: 'yyyy mm dd' },
+        { str: {}, order: 'yyyy mm dd' },
+        { str: [], order: 'yyyy mm dd' },
+        { str: 'invalid', order: 'yyyy mm dd' },
+        { str: '2020 Dec 24', order: 'yyyy mm dd' },
+        { str: '12 24 20', order: 'mm dd yyyy' },
+        { str: '20 12 24', order: 'yyyy mm dd' },
+        { str: '2020 12 24', order: 'yy mm dd' },
+        { str: '12 24 2020', order: 'mm dd yy' }
```

Comment:
Let's have a couple of impossible dates in here like the 30th of February, or the first of "Thirtember", etc.

Reply:
That would definitely make the function fit it's name better. The fact that the existing code has no such feature (at least within the function scope I modified) and was evidently sufficient until now gives me pause. 

I'm gonna spend a little more time looking through the code before implementing this.

Reply:
In case anyone got a notification: my deleted comment was just flat out wrong. Sorry about any confusion I may have caused by deleting it 

These cases will be added in a few minutes.